packages/migration/src/mods/7.0.0/comboboxOnChangedPropsChange.ts

```diff
@@ -0,0 +1,36 @@
+import ts from 'typescript';
+import { IMigrationOptions, migration, ModResult } from '../../migration';
+import { mod } from 'riceburn';
+
+const tagName = 'ComboBox';
+const propName = 'onChanged';
+
+export default migration(
+  'warn ComboBox.onChanged removed',
+  (opts: IMigrationOptions): ModResult[] => {
+    return mod('**/*.tsx', opts).asTypescript((node, modder) => {
+      if (
+        (ts.isJsxOpeningElement(node) || ts.isJsxSelfClosingElement(node)) &&
+        node.tagName.getFullText() === tagName &&
+        node.attributes &&
+        node.attributes.properties
+      ) {
+        const sourceFile = node.getSourceFile();
+        const sourceFileName = sourceFile.fileName;
+        const lineAndCharacter = sourceFile.getLineAndCharacterOfPosition(node.getStart());
+        for (const prop of node.attributes.properties) {
+          if (ts.isJsxAttribute(prop) && prop && prop.name && prop.name.getText() === propName && prop.initializer) {
+            opts.warn(
+              `${sourceFileName}:${lineAndCharacter.line}:${
+                lineAndCharacter.character
+                // tslint:disable-next-line: max-line-length
+              } - ${propName} is removed; please use onChange instead (NOTE: the arguments have changed from ${propName} to onChanged; please update accordingly)`
```

Comment:
Pretty sure this should be as follows:
```suggestion
              } - ${propName} is removed; please use onChange instead (NOTE: the arguments have changed from ${propName} to onChange; please update accordingly)`
```
Also, you could break out of the loop after finding `onChanged`, though given the limited number of ComboBox attributes it's not terribly important.